src/services/pokeService.ts

```diff
@@ -0,0 +1,3 @@
+import { mockPokemonData } from "../mock/pokeData";
```

Comment:
good you separate the service layer from the view layer!

Reply:
👍🏾